src/firstvoices/backend/models/utils/utils.py

```diff
@@ -0,0 +1,60 @@
+import json
+from os import path
+
+from firstvoices.backend.models.category import Category
+
+
+def load_data(json_file):
+    """Helper function to load data from a json file."""
+
+    with open(path.dirname(__file__) + "/" + json_file, encoding="utf-8") as data_file:
+        json_data = json.loads(data_file.read())
+        return json_data
```

Comment:
On the one hand, this totally works. On the other hand, we are already storing some default json values in the AppJson table (default site menu, and the default g2p config). Maybe it would make sense to add this there as well, so it's all together?

Reply:
Looks good to me either way. I believe this separation of logic makes this more flexible, in case we need to change this later or call the categories generation from a mock site generation function. Maybe I can add a ticket to merge this into AppJson later but this works from the MVP perspective.

Reply:
Hmm, I think this way is actually less flexible-- if we ever want to change the default categories we will have to deploy a new file, instead of just updating it in the database

Reply:
Sounds good. I've added a ticket for this in the Backend delighters epic with a brief details about this for us to come back to this.